contrib/gitian-descriptors/gitian-win.yml

```diff
@@ -93,6 +95,9 @@ script: |
     done
   done
   }
+  for i in $HOSTS; do
```

Comment:
It should suffice to add a symlink from ${WRAP_DIR}/${i}/${i}-g++ to /usr/bin/${i}-g++-posix``` instead. Since this is in our path, that means that it would be picked up before the generic one.